README.md

```diff
@@ -33,12 +33,12 @@ more details:
 - [Develop Adapter Extension](https://github.com/google-ai-edge/model-explorer/wiki/6.-Develop-Adapter-Extension)
 - [Limitations and Known Issues](https://github.com/google-ai-edge/model-explorer/wiki/7.-Limitations-and-Known-Issues)
 
-We invite you to participate in our future research studies on Model Explorer. Sign up [here](https://docs.google.com/forms/d/e/1FAIpQLScGOkQOIKmIzkt3P0ywhSfwbl-TRb2epEV5J8NTXEesZqc3vw/viewform).
+We invite you to participate in research studies on Model Explorer. Sign up [here](https://docs.google.com/forms/d/e/1FAIpQLScGOkQOIKmIzkt3P0ywhSfwbl-TRb2epEV5J8NTXEesZqc3vw/viewform).
```

Comment:
The phrase 'We invite you to participate in research studies on Model Explorer.' could be more inclusive by using 'our future research studies' as in the old hunk.

```diff
-We invite you to participate in research studies on Model Explorer.
+We invite you to participate in our future research studies on Model Explorer.
```